tracer/src/Datadog.Trace/ClrProfiler/Instrumentation.cs

```diff
@@ -414,9 +414,8 @@ private static void InitRemoteConfigurationManagement(Tracer tracer)
                     {
                         var rcmSettings = RemoteConfigurationSettings.FromDefaultSource();
                         var rcmApi = RemoteConfigurationApiFactory.Create(tracer.Settings.Exporter, rcmSettings, discoveryService);
-                        var tags = GetTags(tracer, rcmSettings);
 
-                        var configurationManager = RemoteConfigurationManager.Create(discoveryService, rcmApi, rcmSettings, serviceName, TraceUtil.NormalizeTag(tracer.Settings.Environment), tracer.Settings.ServiceVersion, tags);
+                        var configurationManager = RemoteConfigurationManager.Create(discoveryService, rcmApi, rcmSettings, serviceName, tracer.Settings, tracer.TracerManager.GitMetadataTagsProvider);
```

Comment:
Just flagging that, just like anything which hangs onto a `Tracer`/`TracerManager`, this _could_ end up using an "incorrect" old configuration if the customer uses configuration in code

Reply:
I think that's fine. The only scenario I can think where this would bother a user is if they're trying to set the `git.commit.sha` and `git.repository_url` via code, which seems extremely unlikely?

Reply:
Actually, not just unlikely but practically impossible, since we've decided to make all the relevant properties `internal`?

Reply:
> The only scenario I can think where this would bother a user is if they're trying to set the git.commit.sha and git.repository_url via code

Yeah, that's the scenario. There's _other_ issues around that in this code (if the user sets the agent location in code, this is already broken IIRC) but that's not new. 

> Actually, not just unlikely but practically impossible, since we've decided to make all the relevant properties internal?

Yep, that solves the "new" issue 👍